src/database/DataTypes/Requisition.js

```diff
@@ -140,8 +140,12 @@ export class Requisition extends Realm.Object {
     return (this.otherStoreName && this.otherStoreName.name) || '';
   }
 
+  /**
+   * Get all indicators associated with this requisition.
+   * @returns {Array.<ProgramIndicator>}
+   */
   get indicators() {
-    return this.program?.indicators;
+    return this.program?.getActiveindicators();
```

Comment:
`return this.program?.activeIndicators() || []` ? or nah

Reply:
I did consider that, but I think in the case of Realm lists, `null` testing is a bit more consistent than making the getter return two different types (object v. array).